prime-angular-frontend/src/app/modules/site-registration/pages/business-licence/business-licence.component.ts

```diff
@@ -110,6 +138,30 @@ export class BusinessLicenceComponent implements OnInit {
     }
   }
 
+  public downloadBusinessLicence(event: Event) {
+    event.preventDefault();
+    this.siteResource.getBusinessLicenceDocumentToken(this.siteService.site.id)
+      .subscribe((token: string) =>
+        this.utilsService.downloadToken(token)
+      );
+  }
+
+  public toggleDeferred() {
```

Comment:
When you trigger a change on a component it emits up an event for that component.  For a Material slide toggle this is the event `MatSlideToggleChange` which will tell you whether it is checked or not.